tests/qchem/test_hamiltonians.py

```diff
@@ -378,19 +425,6 @@ def circuit(*args):
 
             return circuit
 
-        grad_jax = jax.grad(energy(mol), argnums=0)(*args)
-
-        alpha_1 = np.array(
-            [[3.42425091, 0.62391373, 0.1688554], [3.42525091, 0.62391373, 0.1688554]],
-        )  # alpha[0][0] -= 0.001
-
-        alpha_2 = np.array(
-            [[3.42625091, 0.62391373, 0.1688554], [3.42525091, 0.62391373, 0.1688554]],
-        )  # alpha[0][0] += 0.001
-
-        e_1 = energy(mol)(*[alpha_1])
-        e_2 = energy(mol)(*[alpha_2])
-
-        grad_finitediff = (e_2 - e_1) / 0.002
+        grad_jax = jax.grad(energy(mol), argnums=2)(*args)
 
-        assert np.allclose(grad_jax[0][0], grad_finitediff, rtol=1e-02)
+        assert np.allclose(grad_jax[0][0], 0.02461335393055819, rtol=1e-02)
```

Comment:
Was it being hardcoded like this before as well?

Reply:
[Apparently one could do this instead](https://jax.readthedocs.io/en/latest/notebooks/autodiff_cookbook.html#checking-against-numerical-differences). But I'm lazy and think what exists is good enough as well. Thoughts?